app/models/info_request_batch.rb

```diff
@@ -60,6 +60,12 @@ def create_batch!
         else
           unrequestable << public_body
         end
+        # Sleep between requests in production, in case we're sending a huge
+        # batch which may result in a torrent of auto-replies coming back to
+        # us and overloading the server.
+        if Rails.env.production?
+          sleep 60
+        end
```

Comment:
D'oh, sorry, I missed that (I always think that just creating a request sends it, I'm not sure why).

In your example though, I think we've lost a feature of the previous code where if we have an error creating one request, the transaction will get rolled back and none of the requests will be created or sent. 

Without that, I think we're going to have issues if we ever have to resend a batch, because it won't be marked as sent, but some of the requests might have been?

Is it possible to create the request but stop xapian indexing it until we send the request, so we could move the sleep to the second loop?